ios/FluentUI/Vnext/List/ListTokens.swift

```diff
@@ -102,7 +112,14 @@ class MSFListCellTokens: MSFTokensBase, ObservableObject {
 
     override func updateForCurrentTheme() {
         let currentTheme = theme
-        let appearanceProxy: AppearanceProxyType = currentTheme.MSFListCellTokens
+        let appearanceProxy: AppearanceProxyType
+
+        switch style {
+        case .normal, .none:
+            appearanceProxy = currentTheme.MSFListCellTokens
+        case .persona:
+            appearanceProxy = currentTheme.MSFPersonaTokens
```

Comment:
Using the MSFPersonaTokens here will make the ListCell dependent on the PersonaView. And we want only the PersonaView to be dependent on the ListCell.